pasystem/pasystem-tool/tool/src/webapp/stylesheets/pasystem.css

```diff
@@ -75,38 +75,40 @@
   -o-transition: background-color 0.2s, color 0.2s;
   transition: background-color 0.2s, color 0.2s;
 
-  background-color: #f7e084;
-  color: #000 !important;
+  background-color: var(--bs-warning, #f7e084);
+  color: var(--sakai-brand--darker-7) !important;
```

Comment:
Are these two changes okay in dark mode? Also, do you really need to use !important? We're trying to reduce them if we can and just rely on scoping and ordering to work properly. I'm just asking :)

Reply:
That !important was already there before. Let me check if it is necessary...

Reply:
Removed!